### DISCUSSION TITLE
Prisma Studio

### DISCUSSION THREAD

**Discussion by faustohv** [prisma/prisma#12314] [Q&A] [answered]
From México:

Hi I am Fausto AND wants to use Studio for crud operations for client side with js  scripts by android AND iOS apps?.

Is It posible?

**Comment by austincrim** [comment, marked as answer]
Hey @faustohv 👋 ,

[Prisma Studio](https://www.prisma.io/studio) is an frontend interface to your database. You can definitely do manually CRUD operations through Studio, but it is not meant to be used directly with your application code.

The [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) is a Node.js ORM that can be used with many different backend setups, regardless of whether the frontend is a web application or mobile app.

I hope that helps. Let me know if you have more specific questions!

**Comment by faustohv** [comment]
Dear Austin:

Glad to meet you:

I new on prisma. I want to migrante my php MySQL app to prisma. I have been using xcrud library yo access my database.

In your example the output database récords aré showing in json formato but not in a HTML.

How can I do It If prisma Studio Is not for app AND Is not a spanish tools?

Thanks a lot

Fausto

Get Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
